continuous_integration/environment-mindeps-optional.yaml

```
@@ -0,0 +1,54 @@
+name: test-environment
+channels:
+  - conda-forge
+dependencies:
+  # required dependencies
+  - packaging=20.0
+  - python=3.8
+  - pyyaml=5.3.1
+  - click=7.0
+  - cloudpickle=1.1.1
+  - partd=1.2.0
+  - fsspec=2021.09.0  # required for arrow I/O support
+  - importlib-metadata=4.13.0
+  - toolz=0.10.0  # required for min version of cytoolz
+  # optional dependencies pulled in by pip install dask[array,dataframe]
+  - numpy=1.21
+  - pandas=1.3
+  # optional dependencies used by dask
+  - bokeh=2.4.2
+  - cachey=0.1.1
+  - crick=0.0.3
+  - cytoolz=0.10.1
+  - fastavro=0.22.6
+  - fastparquet=0.7.0
+  - h5py=2.10.0
+  - ipycytoscape=1.0.1
+  - IPython=7.9.0
+  - jinja2=2.10.3
+  - lz4=3.1.1
+  - matplotlib=3.1.1
+  - mimesis=4.0.0
+  - psutil=5.6.3
+  - pyarrow=7.0
+  - python-graphviz=0.8.4
+  - python-snappy=0.5.4  # required for default compression algorithm for to_parquet
+  - s3fs=0.1.3
+  - scikit-image=0.16.2
+  - scipy=1.3.2
+  - sparse=0.12.0
+  - sqlalchemy=1.4.16
+  - tblib=1.5.0
```

`distributed` requires `tblib >= 1.6.0`, so let's update this accordingly. This isn't strictly necessary, but is nice to reduce complexity

```suggestion
  - tblib=1.6.0
```

---

We aren't actually installing distributed in this build - should we? If so, happy to bump these optional dependencies - do you think we should also go ahead and bump the core dependencies that Distributed has a higher minimum version for (i.e. `toolz`, `cloudpickle`, `click`)?

---

Ah, I missed that. Yeah, I think including `distributed` is good to do here. It'd an optional dependency in a similar way that `numpy`, `pandas`, etc. are 

> do you think we should also go ahead and bump the core dependencies that Distributed has a higher minimum version for (i.e. toolz, cloudpickle, click)?

Yeah, let's do that too. Thanks for pointing that out